fe/fe-core/src/main/java/org/apache/doris/common/util/Daemon.java

```diff
@@ -98,10 +98,18 @@ public void setInterval(long intervalMs) {
         this.intervalMs = intervalMs;
     }
 
+    public boolean isStop() {
+        return isStop.get();
+    }
+
```

Comment:
where to use it?

Reply:
This is used when the handler thread is closed